apps/src/applab/applab.js

```diff
@@ -789,10 +773,11 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugWatch: false,
-    localeDirection: studioApp.localeDirection()
+    localeDirection: studioApp.localeDirection(),
+    isDroplet: true
   }));
 
-  Applab.reduxStore.dispatch(changeInterfaceMode(
+  studioApp.reduxStore.dispatch(changeInterfaceMode(
```

Comment:
Maybe overkill, I know.  I guess it depends on whether we expect to use these a lot more, or aggressively discourage accessing the store via studioApp.